crates/types/src/config/common.rs

```diff
@@ -65,8 +65,9 @@ pub struct CommonOptions {
     pub metadata_store_client: MetadataStoreClientOptions,
 
     /// Address to bind for the Node server. Default is `0.0.0.0:5122`
```

Comment:
Maybe? (Depending on which way we go with the default port; see below.)

```suggestion
    /// Address to bind for the Node server. Derived from the advertised address, defaulting
    /// to `0.0.0.0:$PORT` (or `0.0.0.0:5122` if advertised address does not specify port).
```